---
authors: Freek Pols
updated: December 3, 2025
---
# Cappucino

## Introductie
Heb je al eens bedacht dat het verwarmen van melk voor een cappucino eigenlijk heel snel gaat zonder dat de melk verdunt wordt?
De melk wordt ook niet echt plaatselijk verhit door een warmte element.
In plaats daarvan wordt er stoom door de melk geblazen.
De stoom condenseert in de melk en geeft daarbij zijn latente warmte af.
Hierdoor warmt de melk snel op zonder dat er (significant veel) water aan toegevoegd wordt.

```{warning}
Stoom is heel heet en kan voor vervelende brandplekken zorgen.
Wees dus voorzichtig bij het uitvoeren van dit practicum.

Daarnaast, nadat de kookplaat uitgaat is er een onderdruk in de kolf.
Hang dan ook de tube uit het water wanneer je de kookplaat uitzet, anders kan de kolf imploderen.
```

In dit practicum gaan we de verdampingswarmte van water bepalen door middel van een zelfgebouwde cappucino machine.
Het principe is als volgt: We gebruiken de opstelling weergegeven in {numref}`Figuur {number} <fig_cappucino>` waarbij we water verwarmen in een kolf met behulp van een kookplaat.
Door de warmte gaat het water koken en ontstaat er stoom.
De stoom stroomt via een tube naar een afgesloten maatbeker met water.
De stoom condenseert in het maatbeker en geeft daarbij zijn latente warmte af.
Door de temperatuurstijging van het water te meten, evenals de hoeveelheid gram water dat verdampt is, kunnen we de verdampingswarmte van water bepalen.

```{figure} figures/verdampingswarmte_setup.png
:width: 70%
:label: fig_cappucino

Een schematische weergave van de cappucino opstelling.
```

## Theorie

De latente verdampingswarmte van water bedraagt 2257 kJ/kg, dit is veel meer dan de specifieke warmtecapaciteit van water (4.18 kJ/kgK).
Wanneer we de waterdamp (g) door koud water leiden, gaan we er van uit dat de waterdamp condenseert en daarbij zijn latente warmte afgeeft aan het koude water.
Door te bepalen hoeveel gram water verdampt is en hoeveel de temperatuur van het koude water stijgt, kunnen we de latente warmte van verdamping bepalen:

$$ Q_{condensatie} = m_{damp} L = m_{water} c \Delta T $$

met $m_{damp} = \Delta m_{kolf}$.

## Methoden en materialen

### Materialen
- Warmteplaat
- Kolf met stop en tube
- Maatcilinder
- Thermometer
- Weegschaal
- Water


### Procedure

- Vul de kolf met ongeveer 100 mL water: bepaal precies de massa water ($m_{w_1}$).  
- Vul de maatcilinder met ongeveer 100 mL water: bepaal precies de massa water ($m_{bad}$).
- Bepaal de temperatuur van dit waterbad ($T_{bad_1}$).
- Zet de kolf op de warmteplaat en zet de warmteplaat aan - maximale stand 3, zorg ervoor dat de tube goed in het waterbad hangt.
- Wacht tot de temperatuur van het waterbad met ongeveer 20 K is gestegen.
- Stop de meting / zet de warmteplaat uit. Noteer meteen de temperatuur($T_{bad_2}$) en haal de tube uit het water! 

```{warning}
Pas op stoom is heet!
```

- Bepaal precies de massa water in de kolf ($m_{w_2}$).
- Verwerk je resultaten hieronder om de latente warmte van verdamping van water te bepalen.

## Resultaten

In [4]:
import numpy as np

#massas kolf
m_kolf = 0.1152 #kg
m_cilinder = 0.1289

m_w_1 = 0.2029 - m_kolf      # kg
m_bad =  0.1986 - m_cilinder     # kg
T_bad_1 =  17.1 + 273.15   # K

m_w_2 = 0.1972 - m_kolf    # kg
T_bad_2 = 38.7 + 273.15   # K

m_damp = np.abs(m_w_2 - m_w_1)

L = (m_bad * 4186 * (T_bad_2 - T_bad_1)) / m_damp

print(L)


1105632.7578947332


In [5]:
import numpy as np

#massas kolf
m_kolf = 0.1143 #kg
m_cilinder = 0.1291

m_w_1 = 0.2053 - m_kolf      # kg
m_bad =  0.2085 - m_cilinder     # kg
T_bad_1 =  17.2 + 273.15   # K

m_w_2 = 0.1985 - m_kolf    # kg
T_bad_2 = 40.0 + 273.15  # K

m_damp = np.abs(m_w_2 - m_w_1)

L = (m_bad * 4186 * (T_bad_2 - T_bad_1)) / m_damp

print(L)


1114411.6941176476


## Discussie en conclusie
Het experiment is 2 keer uitgevoerd waarbij bij de tweede keer van het uitvoeren we kleinere bubbles door het koelbad lieten gaan, hierbij zijn 2 waarden van $L$ verkregen. De eerste verkregen waarde van L is: $1105632.75 J/kg$, en de tweede verkregen waarde is $1114411.69 J/kg$, terwijl de literatuur spreekt over een waarde van $L= 2257 kJ/kg$, dit toont dat onze vergregen waarden gemiddeld $49.18\% $ van de literatuur waarde afzitten. Dit heeft kunnen gebeuren doordat de stoombellen die door het bad gaan groot en snel omhoog gingen, wat invloed heeft op hoe snel het bad opwarmt en tegelijkertijd ook meer damp nodig heeft om dus het bad met dezelfde temperatuur op te warmen, als zou zijn gebeurd wanneer de bellen veel kleiner waren geweest. Ook had plaatsing van temperatuur meter een invloed kunnen hebben want in eerste experiment zat de thermostaat relatief ver van de opening van de buis die van de kolf naar het bad loopt, ook kan warmteverlies een rol spelen en had er dus damp al kunnen afkoelen in de tranportbuis. Hiermee trekken we de conclusie dat onze gevonden waarden dus strijdig zijn met de literatuurwaarden en als vervolgonderzoek stellen we aan om te gaan onderzoeken hoeveel impact kleine bubbels hebben met de waarde van $L$ te verhogen. 